README.md

```diff
@@ -81,6 +81,8 @@ The configuration supported by the plugin option includes:
 
   * `module`: prefix to be stripped from the `go-gapic-package` used in the generated filenames.
      * _Note: This option is not supported from the Bazel interface._
+
+  * `metadata`: enable generation of [GapicMetadata](https://github.com/googleapis/googleapis/blob/master/gapic/metadata/gapic_metadata.proto) in JSON form.
```

Comment:
Should we say that the default is `false`?

Reply:
Yep, done!